ScriptureRenderingPipeline/Renderers/TranslationManualRenderer.cs

```diff
@@ -50,7 +50,7 @@ public async Task RenderAsync(ZipFileSystem sourceDir, string basePath, string d
                 var templateResult = template.Render(Hash.FromDictionary(new Dictionary<string,object>()
                 {
                     ["content"] = builder.ToString(),
-                    ["contenttype"] = "tw",
+                    ["contenttype"] = "ta",

```

Comment:
I've seen Translation Manuals sometimes have a content type of "ta" and sometimes "tm".  Does this cover both situations?

Reply:
Yeah this is just used to select certain styles in the CSS so at this point it doesn't matter if it is ta or tm